### DISCUSSION TITLE
DEV SESSION [TUESDAY] 26 MAR -- 2030 UTC+8/1330 CET/1230 GMT/ 0830 EST

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#2079] [General]
Where: [via Matrix](https://matrix.to/#/#project-mesa:matrix.org)

Proposed Agenda:
- General updates 
- Dev meeting scheduling pattern: 
    - Second Saturday & Fourth Tuesday of the month? Other suggestions? 
- Google Summer of Code 2024 & NumFOCUS
- Releases 
- MEP Draft
- ?????

Suggested:
- [Make experimental cell spaces work with PropertyGrid #2068](https://github.com/projectmesa/mesa/pull/2068)

Please post suggested agenda items or edit directly. 

As always, please check my times to make sure they are correct.
Note : Here is the notes from the previous session: https://github.com/projectmesa/mesa/discussions/1935

/cc @tpike3, @EwoutH, @Corvince , @wang-boyu ,  @rht, @ankitk50

**Comment by tpike3** [comment]
I recommend adding next release

**Reply by EwoutH** [reply]
Agreed. I would like to have #2068 in.

We could maybe issue a 2.3 release-candidate the week before (Tuesday 19th) and then do the final release on the 26th if no issues pop up.

**Reply by jackiekazil** [reply]
I updated the main list to reflect these.
